app/templates/main/userProfile.html

```diff
@@ -29,12 +29,12 @@ <h6 align="center">{{volunteer.email}}</h6>
 <div class="accordion" id="userProfile">
   <div class="accordion-item">
     <h3 class="accordion-header" id="headingOne">
-      {% set focus = "open" if not visibleAccordion or visibleAccordion == "upcoming" else "collapsed" %}
+      <!-- {% set focus = "open" if not visibleAccordion or visibleAccordion == "upcoming" else "collapsed" %} -->
```

Comment:
Without this piece the Upcoming Events accordion is focused despite it not being the open accordion. Make sure the accordion that is open is always the one that is focused. 
How the page currently looks on page load:
![image](https://user-images.githubusercontent.com/26262045/186759952-41ace136-2865-443e-bdcd-4b2fc6b33373.png)

How the page should look on page load:
![image](https://user-images.githubusercontent.com/26262045/186760042-ad506514-aece-4e12-b76a-f0994aeb2d9f.png)